src/pages/Admin/Room/Edit/RoomEditModal.tsx

```diff
@@ -2,30 +2,40 @@ import React, { useEffect, useState, useCallback, useMemo } from "react";
 import { Form, Modal, Button } from "react-bootstrap";
 import { useForm } from "react-hook-form";
 import ImageInput from "components/atoms/ImageInput";
+import { RoomData_v2 } from "types/rooms";
 
 import { RoomTemplate, ROOM_TEMPLATES } from "settings";
-import { RoomEditModalProps } from "./RoomEditModal.types";
 
 import "./RoomEditModal.scss";
 
+interface RoomEditModalProps {
+  isVisible: boolean;
+  onClickOutsideHandler: () => void;
+  room: RoomData_v2;
+  submitHandler: (values: RoomData_v2, index: number) => void;
+  deleteHandler: () => void;
+}
+
 export const RoomEditModal: React.FC<RoomEditModalProps> = ({
   isVisible,
   onClickOutsideHandler,
   room,
   submitHandler,
   deleteHandler,
 }) => {
-  const [selectedTemplate, setSelectedTemplate] = useState<string>(
-    room.template!
+  const [selectedTemplate, setSelectedTemplate] = useState<string | undefined>(
+    room.template
   );
 
-  const [roomTemplate, setRoomTemplate] = useState<RoomTemplate | null>(null);
+  const [roomTemplate, setRoomTemplate] = useState<RoomTemplate | undefined>(
+    undefined
+  );
```

Comment:
Looks like we can simplify this one too:

```suggestion
  const [roomTemplate, setRoomTemplate] = useState<RoomTemplate>();
```

![image](https://user-images.githubusercontent.com/753891/110567710-0dbd5c00-81a6-11eb-881c-25010bac7c91.png)